htdocs/ticket/list.php

```diff
@@ -364,6 +364,9 @@
 $reshook = $hookmanager->executeHooks('printFieldListFrom', $parameters, $object); // Note that $action and $object may have been modified by hook
 $sql .= $hookmanager->resPrint;
 $sql .= " LEFT JOIN ".MAIN_DB_PREFIX."societe as s ON (t.fk_soc = s.rowid)";
+$sql .= " LEFT JOIN
+	(SELECT MAX(datep) as last_msg_date, fk_element FROM ".MAIN_DB_PREFIX."actioncomm WHERE (code LIKE '%TICKET_MSG%') AND NOT (fk_user_author = 0)  GROUP BY fk_element)
```

Comment:
This subrequest as no link the the parent request meaning that on a huge database of tickets this subrequest will load nearly all database. This may work on small database but will hang on large databases.

The database structure is not yet ready to introduce such a feature. Before being able to report the time since last answer, we should enhance the database structure and code like this:
- First, add a field   "date_last_message"   (other date use the syntax date_...)  of type datetime into the sql of the table llx_ticket.sql and also into the sql migration script.
- Modify the code to update this field in llx_ticket each time a new message is set on a ticket.
This must be done into another PR. Once this PR is validated, then you will be able to modify this request to use the new field to find the 'a.last_msg_date'

Reply:
Thanks for the review !